### PR TITLE
add method getPersonnelSearch 

### PR DESCRIPTION
**Pull request by franzelunico**

### CODE REVIEW

File: src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionSSID.java [hunk 1]
```diff
@@ -261,7 +261,8 @@ public void validarComponenteImagenFormularioRegistroEquipamiento() throws Throw
 
     @And("^seleccionar submenu 'Personal' en menu 'Personal'$")
     public void seleccionarSubMenuPersonal() throws Throwable {
-        personnelSearch = menuPersonal.selectSubMenuPersonnel();
+        Assert.assertTrue(menuPersonal.selectSubMenuPersonnel());
```
Comment (marbusjim) [inline src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionSSID.java]: add wrong message in your assertion

File: src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionSSID.java [hunk 2]
```diff
@@ -261,7 +261,8 @@ public void validarComponenteImagenFormularioRegistroEquipamiento() throws Throw
...
+        personnelSearch = menuPersonal.getPersonnelSearch();
         Assert.assertTrue(personnelSearch.validateInputFindPersonIsVisible());
```
Comment (marbusjim) [inline src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionSSID.java]: add wrong message in your assertion

**Review by marbusjim — COMMENTED** [review]
with comments

**Review by ctola91 — APPROVED** [review]

**Review by ivanMise — APPROVED** [review]

**Review by danielrpg — APPROVED** [review]
Aproved

**Pull request merged**